src/components/icon/icon.test.ts

```diff
@@ -154,4 +154,79 @@ describe('<sl-icon>', () => {
       expect(ev).to.exist;
     });
   });
+
+  describe('svg spritesheets', () => {
+    //  For some reason ESLint wants to fail in CI here, but works locally.
+    /* eslint-disable */
+    it('Should properly grab an SVG and render it from bootstrap icons', async () => {
+      registerIconLibrary('sprite', {
+        resolver: name => `/docs/assets/images/sprite.svg#${name}`,
+        mutator: svg => svg.setAttribute('fill', 'currentColor'),
+        spriteSheet: true
+      });
+
+      const el = await fixture<SlIcon>(html`<sl-icon name="arrow-left" library="sprite"></sl-icon>`);
+
+      await elementUpdated(el);
+
+      const svg = el.shadowRoot?.querySelector("svg[part='svg']");
+      const use = svg?.querySelector(`use[href='/docs/assets/images/sprite.svg#arrow-left']`);
+
+      expect(svg).to.be.instanceof(SVGElement);
+      expect(use).to.be.instanceof(SVGUseElement);
+
+      // This is kind of hacky...but with no way to check "load", we just do a timeout :shrug:
```

Comment:
Should we start a convention for things like this? I usually use `TODO`. 🤷🏻‍♂️

Reply:
Probably....works for me.